doc/operation-and-maintenance/known-issues.md

```diff
@@ -10,3 +10,16 @@ MongooseIM will not connect to MySQL over TLS on OTP 20.3 due to [the MySQL driv
 * Upgrade OTP to 21.0 or higher.
 * Use unencrypted communication with MySQL.
 
+## MSSQL connectivity via ODBC
+
+ODBC driver currently used by MongooseIM is known to work only with Ubuntu Xenial x64 and FreeTDS 0.91-6.1build1.
+
+It definitely doesn't work on Ubuntu Bionic, CentOS 7 and macOS Mojave with the latest FreeTDS version.
```

Comment:
I'm not that sure about the harshness of `definitely` there. I'd rather go for `We've tested negatively on...` or simply omitting that `definitely` from there.

Reply:
It does not currently work on Ubuntu Bionic, CentOS 7 and macOS Mojave with the latest FreeTDS version.